### ISSUE TITLE
Add config option to disable horizon

### ISSUE THREAD

**Issue by Quent42340** [minetest/minetest#7975]
##### Issue type
 - Feature request
 
##### Summary

Since #7932, stars are hidden behind horizon, like moon and sun.

However, in skyblock/space games, stars, sun and moon have to be rendered no matter where they are.

So a server-side option to disable horizon could be really nice.

**Comment by paramat** [comment]
I agree. I almost did this in a recent commit but it would have been hacky and rushed, it needs to be done properly, perhaps as part of sky API improvements (@Jordach ?).

What needs removing:
The NSEW horizons and sky base drawn after everything else.
The sun horizon glow texture.
The NSEW 'far cloudy fog things' in the case of deep space, possibly also for a groundless skyblock world.

Currently sun and moon are not drawn for their full orbit, so will need to be.

**Comment by sfan5** [comment]
Did this get implemented in the meantime, by chance?

**Comment by Wuzzy2** [comment]
No.

**Comment by paramat** [comment]
Erm i suspect the new set_sky API makes this possible by using one of the special sky modes, not sure.

**Comment by Wuzzy2** [comment]
No, it doesn't.

**Comment by paramat** [comment]
Ok. As i wrote i was not sure, no need for a downvote =)

**Comment by sfan5** [comment]
Did this get implemented in the meantime, by chance?

**Comment by GreenXenith** [comment]
`set_sky` allows one to set the sky and horizon colors the same, "hiding" the horizon. But I am not certain it is perfect due to other color modifications (edge darkening, sunrise bleeding). In other words, not really.
